packages/contracts/contracts/strategies/HackMoneyStrategy.sol

```diff
@@ -339,6 +345,7 @@ contract HackMoneyStrategy is HackMoneyStrategyBase, IHackMoneyStrategy {
     /**
      * @dev use premium in strategy to reduce position size if collateral ratio is out of range
      */
+    // TODO: Remove this
```

Comment:
Is there any cases where we would want to reduce position? In the current definition of strategy Id say no, but would it be an "emergency" case where we would want to get out?

Reply:
What is this "emergency" case? We might want to stop most of the actions and let users withdraw assets

Reply:
yes that's something to think about later, maybe worth checking what's the security measures of Ribbon